src/reducer.test.js

```diff
@@ -0,0 +1,34 @@
+import reducer from './reducer';
+
+import {
+  setRegions,
+} from './actions';
+
+const initialState = {
+  regions: [],
+};
+
+describe('reducer', () => {
+  context('with action matched', () => {
+    describe('setRegions', () => {
+      it('changes Regions array', () => {
+        const regions = [
+          { id: 1, name: '서울' },
+        ];
+
+        const state = reducer(initialState, setRegions(regions));
+
+        expect(state.regions).not.toHaveLength(0);
```

Comment:
```suggestion
        expect(state.regions).toHaveLength(1);
```

전달한 상태의 배열 내부 요소가 1개이기 때문에 명확히 표현해주는 게 좋겠습니다.